README.md

```diff
@@ -19,66 +25,56 @@ All the files refers to the `SLUS-00067` version of the game.
+  * `sudo apt-get install gcc-mipsel-linux-gnu binutils-mips-linux-gnu`
+  * `pip install spimdisasm tqdm intervaltree`
+  * inside the folder of your choice `git clone https://github.com/Xeeynamo/sotn-decomp.git`
+  * `git submodule update --init`
+  * inside the newly created repo, create a new `iso/` folder, and extract the content of the game disc
+
+* build
+  * run `make extract` to generate the assembly files in the `asm/` directory
+  * run `make all` to compile the binaries in the `build/` directory
+you don’t need to use it now but note that `make clean` deletes the build directory
+Some non-matching functions are present in the source preprocessed by the macro `NON_MATCHING`. You can still compile the game binaries by running `CPP_FLAGS=-DNON_MATCHING make`. In theory they might be logically equivalent in-game, but I cannot promise that. Few of them could match by tuning or changing the compiler.
+
+* decompile
+  * after setup and build, choose an overlay (eg. `ST/WRP`)
+  * Look for one of those function which hasn't successfully decompiled yet (eg. `INCLUDE_ASM("asm/st/wrp/nonmatchings/6FD0", func_801873A0);`)
+  * Look for its assembly file (eg. `asm/st/wrp/nonmatchings/6FD0/func_801873A0.s`)
+  * Run `SOURCE=src/st/wrp/6FD0.c make ctx.c` then `ASSEMBLY=asm/st/wrp/nonmatchings/6FD0/func_801873A0.s make decompile` to dump the decompiled code into ctx.c.m2c
+Replace the `INCLUDE_ASM(...);` you targeted with the content of `ctx.c.m2c` and tweak the code with `asm-differ`
+You will probably have some differences from your compiled code to the original; keep refactoring the code and move variables around until you have a 100% match.
 
 There are a few tricks to make the process more streamlined:
-* Use [decomp.me](https://decomp.me/new) with GCC 2.7.2 for PS1. Be aware that the repo is using GCC 2.6.x, so decomp.me will sometimes give a slightly different output
-* Use [decomp-permuter](https://github.com/simonlindholm/decomp-permuter) to [solve some mismatches](https://github.com/mkst/esa/wiki#decomp-permuter)
+* Use [decomp.me](https://decomp.me/) with GCC 2.7.2 for PS1. Be aware that the repo is using GCC 2.6.x, so decomp.me will sometimes give a slightly different output. 
+* the “context” section of decomp.me, is provided by the cmd `SOURCE=src/st/wrp/6FD0.c make ctx.c` as mentionned in the how to decompile.
+* Use [decomp-permuter](https://github.com/simonlindholm/decomp-permuter) to solve some mismatches
 * Use [this](https://github.com/mkst/sssv/wiki/Jump-Tables) and [this](https://github.com/pmret/papermario/wiki/GCC-2.8.1-Tips-and-Tricks) guide to understand how some compiler patterns work
 * Use the `#ifndef NON_MATCHING` if your code is logically equivalent but you cannot yet fully match it
 
-# Notes
 
-* I suspect that GCC 2.6.x / PSY-Q 3.4 have been used to originally compile `DRA.BIN`
-* `main.exe` uses PS-X libraries that might have been created with a different compiler and with `-O1` rather than `-O2`
+## Ressources:
```

Comment:
🔕 Resources, with one S. I realise that for French-speaking people this might caught off guard!